src/main/java/com/conveyal/r5/profile/PerTargetPropagater.java

```diff
@@ -35,15 +35,16 @@
     /** the profilerequest (used for walk speed etc.) */
     public final ProfileRequest request;
 
-    public PerTargetPropagater (int[][] travelTimesToStopsEachIteration, int[] nonTransferTravelTimesToTargets, LinkedPointSet targets, ProfileRequest request, int cutoffSeconds) {
+    public PerTargetPropagater (int[][] travelTimesToStopsEachIteration, int[] nonTransitTravelTimesToTargets, LinkedPointSet targets, ProfileRequest request, int cutoffSeconds) {
         this.travelTimesToStopsEachIteration = travelTimesToStopsEachIteration;
-        this.nonTransferTravelTimesToTargets = nonTransferTravelTimesToTargets;
+        this.nonTransitTravelTimesToTargets = nonTransitTravelTimesToTargets;
         this.targets = targets;
         this.request = request;
         this.cutoffSeconds = cutoffSeconds;
     }
 
-    public void propagate (Reducer reducer) {
+    private void propagate (Reducer reducer, TravelTimeReducer travelTimeReducer) {
```

Comment:
This signature is strange, where only one of the two parameters should be passed in and one overrides the other. If there's no other reasonable way to do this, there should at least be Javadoc explaining which parameter takes priority over the other, and why there are two different ones. The more standard way would be to overload the propagate() name with two different parameter types but I guess that just pushes the problem farther down the stack.